.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        version:
+          - '1.0'
+          - '1.3'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v1
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - uses: julia-actions/julia-buildpkg@latest
+      - uses: julia-actions/julia-runtest@latest
+      - uses: julia-actions/julia-uploadcodecov@latest
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+  docs:
+    name: Documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: '1.0'
+      - run: julia --project=docs -e '
+          using Pkg;
+          Pkg.develop(PackageSpec(; path=pwd()));
+          Pkg.instantiate();'
+      - run: julia --project=docs docs/make.jl
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
IIRC you need to add `DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}` here too.

https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#GitHub-Actions-1
https://github.com/JuliaDocs/Documenter.jl/issues/1177

You can add the secret at
https://github.com/c42f/TerminalLoggers.jl/settings/secrets

Reply:
Thanks I've tried adding that workaround.

Reply:
It just occurred to me that, if you are going to run test CI on GitHub Actions, you need an extra configuration (i.e., using a custom token [like this in `.github/workflows/CompatHelper.yml`](https://github.com/tkf/BenchmarkCI.jl/pull/24/commits/87fa8d1a51435cbc3bbd28de693504367981f631)) if you want CompatHelper.jl to trigger the CI. Ref: https://github.com/bcbi/CompatHelper.jl/issues/176

Then, if you do that, I think it makes sense to use the same token for `GITHUB_TOKEN` you pass to Documenter, instead of using `DOCUMENTER_KEY`. Ref: https://github.com/JuliaDocs/Documenter.jl/issues/1177#issuecomment-580073032

Reply:
So that would be a *personal* github access token rather than an autogenerated one for the "github action role"?

Reply:
Yes. I think you can also create a "bot account" and use the token of it.

Reply:
Cool thanks for explaining. I'll keep that in mind if we need to add some more actions.